docs/source/conf.py

```diff
@@ -33,6 +33,7 @@
 extensions = [
     "sphinx.ext.autodoc",
     "sphinx_rtd_theme",
+    "myst_parser",
```

Comment:
What's that?

Reply:
This is needed for rendering Markdown. I took this from the documentation. Link is in the commit message